app/views/devise/sessions/new.html.erb

```diff
@@ -3,6 +3,9 @@
     <div class="govuk-grid-column-two-thirds">
       <h1 class="govuk-heading-l">Sign in to your account to submit CORE data</h1>
 
+      <% flash_to_model_errors(resource) %>
+      <%= f.govuk_error_summary %>
+
```

Comment:
The error summary should appear above the `h1`. We also need to show error messages on the respective fields that have been validated.

Reply:
Moved the errors above the header and updated the screenshot. 

I'd say you generally shouldn't add the error to specific fields because it allows malicious attackers to check whether an email exists in the system or not, which is not usually recommended?

Reply:
If I don’t enter any value into any field, I’d expect validation to tell me to enter an email address and password.

If I enter an email in the incorrect format, I’d expect validation to tell me the input is not using a valid email format.

That sort of thing.

Reply:
Added those two